packages/block-library/src/columns/block.json

```diff
@@ -49,7 +49,9 @@
 		"layout": {
 			"allowSwitching": false,
 			"allowInheriting": false,
-			"allowEditing": false,
+			"allowEditing": true,
+			"allowJustification": true,
+			"allowVerticalAlignment": false,
```

Comment:
As we are adding vertical alignment using BlockControls we don't need to show this.

```js
<BlockControls>
				<BlockVerticalAlignmentToolbar
					onChange={ updateAlignment }
					value={ verticalAlignment }
				/>
			</BlockControls>
```

Reply:
Also if we remove that then vertical alignment will not be applied to individual column.